apps/met-airquality/server/main.js

```diff
@@ -185,6 +185,56 @@ const doPlotParams = function () {
                 displayPriority: 1,
                 displayGroup: 2
             });
+
+        var optionsMap = {
+            'Fcst lead time': "select ld.fcst_lead as xVal, ",
+            'Threshold': "select h.fcst_thresh as xVal, ",
+            'Valid UTC hour': "select unix_timestamp(ld.fcst_valid_beg)%(24*3600)/3600 as xVal, ",
+            'Init UTC hour': "select unix_timestamp(ld.fcst_init_beg)%(24*3600)/3600 as xVal, ",
+            'Valid Date': "select unix_timestamp(ld.fcst_valid_beg) as xVal, ",
+            'Init Date': "select unix_timestamp(ld.fcst_init_beg) as xVal, "
+        };
+
+        matsCollections.PlotParams.insert(
+            {
+                name: 'x-axis-parameter',
+                type: matsTypes.InputTypes.select,
+                options: Object.keys(optionsMap),
+                optionsMap: optionsMap,
+                selected: '',
+                controlButtonCovered: true,
+                unique: false,
+                default: Object.keys(optionsMap)[4],
+                controlButtonVisibility: 'block',
+                displayOrder: 9,
+                displayPriority: 1,
+                displayGroup: 2,
+            });
+
+        optionsMap = {
+            'Fcst lead time': "ld.fcst_lead as yVal, ",
+            'Threshold': "h.fcst_thresh as yVal, ",
+            'Valid UTC hour': "unix_timestamp(ld.fcst_valid_beg)%(24*3600)/3600 as yVal, ",
+            'Init UTC hour': "unix_timestamp(ld.fcst_init_beg)%(24*3600)/3600 as yVal, ",
+            'Valid Date': "unix_timestamp(ld.fcst_valid_beg) as yVal, ",
+            'Init Date': "unix_timestamp(ld.fcst_init_beg) as yVal, "
```

Comment:
Is this chunk missing the `select` keyword? It looks like L189-L196 uses `select`. I also wonder if it'd make sense to have a `const xOptionsMap` and `const yOptionsMap` so it's a bit more explicit that these are supposed to be different and aren't supposed to be modified.

Also - I wonder if we should start preferring `const` and `let` over `var`. `let` is similar to `var` but only scoped to the block where it's declared whereas `var` expands out to be available to the entirety of the function or to a global variable. 

MDN docs:
* [`var`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var)
* [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)